conferences/2018/javascript.json

```diff
@@ -150,6 +150,15 @@
     "twitter": "@fronttrends"
   },
   {
+    "name": "JSConf EU",
+    "url": "https://2018.jsconf.eu/",
+    "startDate": "2018-06-02",
+    "endDate": "2018-06-03",
+    "city": "Berlin",
+    "country": "Germany",
+    "twitter": "@jsconfeu",
```

Comment:
You have to remove the coma here. `JSON` files don't support coma dangling. 😉

Reply:
Good catch, I just copy pasted from the template.
I also applied the fix there.

Reply:
Awesome thanks!